packages/vuetify/src/directives/intersect/index.ts

```diff
@@ -5,12 +5,12 @@ import type {
 } from 'vue'
 
 type ObserveHandler = (
+  isIntersecting: boolean,
```

Comment:
Not *really* related but the original order of this makes no sense, most of the time you only care about `isIntersecting` so it should be the first argument.